v2/mongodb-to-googlecloud/pom.xml

```diff
@@ -29,13 +29,16 @@
 
     <artifactId>mongodb-to-googlecloud</artifactId>
 
+    <properties>
+    <google-truth.version>1.0.1</google-truth.version>
+    </properties>
+
```

Comment:
```suggestion
```

Reply:
Should be using v2/pom.xml for truth version anyway. Removed.